.github/workflows/linters.yml

```diff
@@ -51,6 +51,7 @@ jobs:
       - name: Environment Information
         run: npx envinfo
       - name: Get release version numbers
+        if: ${{ github.event.pull_request.base.ref == github.event.pull_request.base.default_branch }}
```

Comment:
What does GitHub do here then if the condition fails? Does it skip the entire job, or just this step, in which case what is `steps.get-released-versions.outputs.NODE_RELEASED_VERSIONS` going to evaluate to in the next step?

Reply:
it skips only this step

Reply:
It skips the step where it parses the changelogs and validate the version number baed on the pattern only – see https://github.com/nodejs/node/runs/2123211599 for an example.